data/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class DataConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "data"
```

Comment:
```suggestion
    name = "data"
    app_label = "data"
```

voir si besoin pour Max ou pas